airbyte/_processors/sql/base.py

```diff
@@ -435,9 +434,6 @@ def _create_table(
         column_definition_str: str,
         primary_keys: list[str] | None = None,
     ) -> None:
-        if DEBUG_MODE:
-            assert table_name not in self._get_tables_list(), f"Table {table_name} already exists."
-
```

Comment:
This, and the below, were significantly slowing down execution, at the cost of approx. 30 seconds per call to _get_tables_list(). Removed here, and also below.